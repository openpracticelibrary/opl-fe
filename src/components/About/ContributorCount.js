import React from "react";
import { CircularProgress } from "@material-ui/core";
import { useQuery } from "@apollo/react-hooks";
import ComponentQueryError from "../shared/QueryState/ComponentQueryError";
import { GET_CONTRIBUTORS } from "../../graphql";

const ContributorCount = ({ color }) => {
  const { loading, error, data } = useQuery(GET_CONTRIBUTORS);
  if (loading) return <CircularProgress color={color} size="10rem" />;
  if (error) return <ComponentQueryError err={error} />;

  return (
    <div data-testid="contributorNum">
      {data.usersConnection.aggregate.totalCount}
    </div>
  );
};

export default ContributorCount;
