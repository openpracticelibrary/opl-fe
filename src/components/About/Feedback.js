import React from 'react';
import { makeStyles } from '@material-ui/core/styles/index';
import Box from '@material-ui/core/Box';
import { Grid, Button } from "@material-ui/core";
import Typography from "@material-ui/core/Typography";
import { navigate } from "@reach/router";
import GrayRain from "../../assets/icons/rain.svg";
import FeedbackIcon from "../../assets/icons/about-feedback.png"

const useStyles = makeStyles((theme) => ({
  feedbackBox: {
    display: "flex",
    alignContent: "flex-start",
    textAlign: "left",
    width: "80%",
    flexWrap: "wrap",
  },
  feedback: {
    display: "flex",
    justifyContent: "center",
    alignContent: "center",
    alignItems: "center",
    backgroundImage: `linear-gradient(0deg, #FFFFFF 0%, rgba(255,255,255,0.95) 71.03%, rgba(255,255,255,0.85) 100%), url(${GrayRain})`,
    width: "100%",
    height: "500px"
  },
  feedbackButton: {
    borderRadius: "32px",
    width: "15rem",
    backgroundColor: "#ffffff",
    marginTop: theme.spacing(5),
    padding: theme.spacing(2),
    borderColor: theme.palette.common.discovery_blue,
    borderWidth: "1px",
    borderStyle: "solid",
  },
  learn: {
    textAlign: "left",
    fontWeight: "normal",
    paddingTop: theme.spacing(6),
  },
  feedbackIcon: {
    display: "flex",
    alignItems: "flex-end",
    alignContent: "flex-end",
    justifyContent: "flex-end",
    padding: theme.spacing(0, 4),
  },
}));

const Feedback = (props) => {
  const classes = useStyles();

  return (
    <Grid
      container
      direction="row"
      className={classes.feedback}
    >
      <Grid item xs={1}/>
      <Grid item xs={1}>
        <Box className={classes.feedbackIcon}>
          <img src={FeedbackIcon} alt="Feedback" />
        </Box>
      </Grid>
      <Grid item xs={7}>
        <Box className={classes.feedbackBox}>
          <Box>
            <Typography variant={"h1"} data-testid="feedbackMsg">
              {props.header}
            </Typography>
            <Typography variant={"h5"} className={classes.learn}>
              {props.subheader}
            </Typography>
          </Box>
        </Box>
      </Grid>
      <Grid item xs={3}>
        <Box>
          <Button
            variant="contained"
            disableElevation
            className={classes.feedbackButton}
            onClick={() => navigate(props.buttonTarget)}
          >
            {props.buttonText}&nbsp;{props.buttonIcon}
          </Button>
        </Box>
      </Grid>
    </Grid>
  )
};

export default Feedback;
