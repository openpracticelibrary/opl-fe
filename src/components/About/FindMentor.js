import React from 'react';
import { makeStyles } from '@material-ui/core/styles/index';
import Grid from "@material-ui/core/Grid";
import Typography from "@material-ui/core/Typography";
import Link from "@material-ui/core/Link";

const useStyles = makeStyles((theme) => ({
  teamHeader: {
    display: "flex",
    flexDirection: "row",
    justifyContent: "space-between",
    paddingTop: theme.spacing(12),
    paddingBottom: theme.spacing(6),
  },
  paraSpacer: {
    display: "flex",
    flexDirection: "row",
    justifyContent: "space-between",
    paddingTop: theme.spacing(6),
    paddingBottom: theme.spacing(10),
  },
}));

const FindMentor = (props) => {
  const classes = useStyles();

  return (
    <>
      <Grid
        data-testid="pageGrid"
        container
        direction="row"
        className={classes.pageWrapper}
      >
        <Grid item xs={1}/>
        <Grid item xs={11} className={classes.teamHeader} data-testid="findMentor">
          <Typography variant={"h1"}>
            {props.header}
          </Typography>
        </Grid>
        <Grid item xs={1}/>
        <Grid item xs={4}>
          <Typography variant={"body1"}>
            {props.bodyPara}
          </Typography>
        </Grid>
        <Grid item xs={1}/>
        <Grid item xs={6}>
          <Link href={props.linkTarget}>
            <img src={props.logo} alt="Logo" width="400rem"/>
          </Link>
        </Grid>
        <Grid item xs={1}/>
        <Grid item xs={11} className={classes.paraSpacer}>
          <Link href={props.linkTarget}>
            <Typography variant="body1">
              {props.linkIcon}&nbsp;&nbsp;{props.linkText}
            </Typography>
          </Link>
        </Grid>
      </Grid>
    </>
  )
};

export default FindMentor;
