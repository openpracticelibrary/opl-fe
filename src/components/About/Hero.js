import React from 'react';
import {
  Box,
  Typography
} from '@material-ui/core';
import GrayDiagonals from "../../assets/icons/graylines.svg";
import ExploreStickies from "../../assets/icons/explorestickies.png";

const Hero = (props) => (
  <Box
    display="flex"
    alignContent="center"
    alignItems="center"
    justifyContent="space-around"
    style={{
      backgroundImage: `linear-gradient(180.18deg, #F3F3F3 0%, #FFFFFF 13.64%, #FFFFFF 43.94%, rgba(255,255,255,0.94) 79.58%, rgba(255,255,255,0.62) 100%), url(${GrayDiagonals})`
    }}
    height="28rem"
    width={1}
  >
    <Box>
      <Typography
        component="h1"
        variant="h3"
        gutterBottom
        data-testid="hellotext"
      >
        {props.greeting}
      </Typography>
      <Typography
        variant={"h5"}
        gutterBottom
        data-testid="welcomeText"
      >
        {props.welcomeMessage}
      </Typography>
    </Box>
    <Box>
      <img src={ExploreStickies} alt="About Us" width="100%" />
    </Box>
  </Box>
);

export default Hero;
