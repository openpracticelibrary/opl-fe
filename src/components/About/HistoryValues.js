import React from 'react';
import { makeStyles } from '@material-ui/core/styles/index';
import Box from '@material-ui/core/Box';
import Grid from "@material-ui/core/Grid";
import Typography from "@material-ui/core/Typography";

const useStyles = makeStyles((theme) => ({
  pageWrapper: {
    backgroundColor: theme.palette.common.true_white,
  },
  historyValuesHeader: {
    display: "flex",
    flexDirection: "row",
    justifyContent: "space-between",
    paddingBottom: theme.spacing(3),
  },
  valuesBox: {
    display: "flex",
    justifyContent: "center",
    paddingBottom: theme.spacing(2),
  },
  attributeBox: {
    display: "flex",
    alignItems: "center",
    flexFlow: "column",
    justifyContent: "center",
    textAlign: "center",
    boxSizing: "border-box",
    height: "340px",
    width: "330px",
    borderRadius: "50%",
    borderStyle: "solid",
    borderColor: "#00cc99",
    borderWidth: "1px",
    boxShadow: "2px 2px 8px 0 rgba(0,0,0,0.1)",
    padding: theme.spacing(5),
    margin: theme.spacing(2),
  },
  circleHeader: {
    fontFamily: "Quicksand",
    fontSize: "40px",
    fontWeight: "normal",
    textAlign: "center",
    lineHeight: "1.67",
    color: theme.palette.common.discovery_blue,
  },
  circleBody: {
    fontFamily: "Quicksand",
    fontSize: "16px",
    fontWeight: "normal",
    textAlign: "center",
    lineHeight: "1.33",
    color: "#9fa4a3",
  },
  paraBody: {
    paddingBottom: theme.spacing(2),
  }
}));

const HistoryValues = (props) => {
  const classes = useStyles();

  return (
    <>
      <Grid
        data-testid="pageGrid"
        container
        direction="row"
        className={classes.pageWrapper}
      >
        <Grid item xs={1}/>
        <Grid item xs={11} className={classes.historyValuesHeader} data-testid="historyValuesHeader">
          <Typography variant={"h1"}>
            {props.header}
          </Typography>
        </Grid>
        <Grid item xs={1}/>
        <Grid item xs={11}>
          <Typography variant={"body1"} className={classes.paraBody}>
            {props.bodyPara1}
          </Typography>
          <Box className={classes.valuesBox}>
            <Box className={classes.attributeBox}>
              <img src={props.circleIcon1} alt={props.circleTitle1}/>
              <Typography className={classes.circleHeader}>{props.circleTitle1}</Typography>
              <Typography className={classes.circleBody}>{props.circleText1}</Typography>
            </Box>
            <Box className={classes.attributeBox}>
              <img src={props.circleIcon2} alt={props.circleTitle2}/>
              <Typography className={classes.circleHeader}>{props.circleTitle2}</Typography>
              <Typography className={classes.circleBody}>{props.circleText2}</Typography>
            </Box>
            <Box className={classes.attributeBox}>
              <img src={props.circleIcon3} alt={props.circleTitle3}/>
              <Typography className={classes.circleHeader}>{props.circleTitle3}</Typography>
              <Typography className={classes.circleBody}>{props.circleText3}</Typography>
            </Box>
          </Box>
          <Typography variant={"body1"}>
            {props.bodyPara2}
          </Typography>
        </Grid>
      </Grid>
    </>
  )
};

export default HistoryValues;
