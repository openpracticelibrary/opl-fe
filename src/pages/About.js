import React from 'react';
import Box from "@material-ui/core/Box";
import Hero from "../components/About/Hero";
import HistoryValues from "../components/About/HistoryValues";
import ProductTeam from "../components/About/ProductTeam";
import ProcessModel from "../components/About/ProcessModel";
import FindMentor from "../components/About/FindMentor";
import Feedback from "../components/About/Feedback";

export default function About(props) {
  return (
    <Box
      data-testid="aboutPageGrid"
      display="flex"
      flexDirection="column"
    >
      <Hero
        greeting="Learn about us"
        inquiry="Sound up your alley?"
        navigate={props.navigate}
      />
      <Box
        maxWidth="80rem"
        alignSelf="center"
        mx={6}
      >
        <HistoryValues />
        <ProductTeam />
        <ProcessModel />
        <FindMentor />
        <Feedback />
      </Box>
    </Box>
  );
};
