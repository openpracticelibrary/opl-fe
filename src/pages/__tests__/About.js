import React from "react";
import { cleanup, render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { MockedProvider } from "@apollo/react-testing";
import renderer from 'react-test-renderer';
import About from '../About';

afterEach(cleanup);

it("renders the About page without crashing", () => {
  const { getByTestId } = render(
    <MockedProvider mocks={[]}>
      <About />
    </MockedProvider>
  );

  expect(getByTestId("aboutPageGrid")).toHaveTextContent("Learn about us");
});

it("matches the About page snapshot", () => {
  const tree = renderer.create(
    <MockedProvider mocks={[]}>
      <About />
    </MockedProvider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
