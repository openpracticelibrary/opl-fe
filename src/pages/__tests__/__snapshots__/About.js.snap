// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the About page snapshot 1`] = `
<div
  className="MuiGrid-root makeStyles-pageWrapper-245 MuiGrid-container MuiGrid-direction-xs-column"
  data-testid="aboutPageGrid"
>
  <div
    className="MuiGrid-root makeStyles-titleBox-351 MuiGrid-container"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
    />
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5"
    >
      <div
        className="MuiBox-root MuiBox-root-357 makeStyles-textBox-353"
      >
        <h1
          className="MuiTypography-root makeStyles-hello-354 MuiTypography-h1"
          data-testid="hellotext"
        >
          Learn about us
        </h1>
        <h5
          className="MuiTypography-root makeStyles-welcome-355 MuiTypography-h5"
          data-testid="welcomeText"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
        </h5>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
    >
      <img
        alt="About Us"
        height="auto"
        src="explorestickies.png"
        width="600px"
      />
      <div
        className="MuiBox-root MuiBox-root-388 makeStyles-loginBox-352"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-loginButton-391"
          data-testid="loginButton"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            <span
              className="MuiTypography-root makeStyles-buttonText-392 MuiTypography-button"
            >
              Login
            </span>
          </span>
        </button>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-content-246 MuiGrid-container MuiGrid-spacing-xs-1"
  >
    <div
      className="MuiBox-root MuiBox-root-446 makeStyles-contentBox-247"
    >
      <div
        className="MuiGrid-root makeStyles-pageWrapper-447 MuiGrid-container"
        data-testid="pageGrid"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-historyValuesHeader-448 MuiGrid-item MuiGrid-grid-xs-11"
          data-testid="historyValuesHeader"
        >
          <h1
            className="MuiTypography-root MuiTypography-h1"
          >
            Our history & values
          </h1>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root makeStyles-paraBody-453 MuiTypography-body1"
          >
            [3 years and 7 months ago], Red Hat Open Innovation Labs saw an opportunity to document the open practices and principles they used during customer engagements, thus was born the Open Practice Library. That vision turned into a website of Devops and product development practices for running workshops and learning about the discovery and delivery of software.
          </p>
          <div
            className="MuiBox-root MuiBox-root-454 makeStyles-valuesBox-449"
          >
            <div
              className="MuiBox-root MuiBox-root-455 makeStyles-attributeBox-450"
            >
              <img
                alt="Friendly"
                src="about-friendly.png"
              />
              <p
                className="MuiTypography-root makeStyles-circleHeader-451 MuiTypography-body1"
              >
                Friendly
              </p>
              <p
                className="MuiTypography-root makeStyles-circleBody-452 MuiTypography-body1"
              >
                We are here to be friends with and make connections for the Open Practice Library community.
              </p>
            </div>
            <div
              className="MuiBox-root MuiBox-root-456 makeStyles-attributeBox-450"
            >
              <img
                alt="Trustworthy"
                src="about-trustworthy.png"
              />
              <p
                className="MuiTypography-root makeStyles-circleHeader-451 MuiTypography-body1"
              >
                Trustworthy
              </p>
              <p
                className="MuiTypography-root makeStyles-circleBody-452 MuiTypography-body1"
              >
                We strive to provide expertise and advice that will seamless fit into our users work life.
              </p>
            </div>
            <div
              className="MuiBox-root MuiBox-root-457 makeStyles-attributeBox-450"
            >
              <img
                alt="Inspiring"
                src="about-inspiring.png"
              />
              <p
                className="MuiTypography-root makeStyles-circleHeader-451 MuiTypography-body1"
              >
                Inspiring
              </p>
              <p
                className="MuiTypography-root makeStyles-circleBody-452 MuiTypography-body1"
              >
                We will venture into the unknown and share experiments that will drive innovation of the library.
              </p>
            </div>
          </div>
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            Fast forward through a few iterations and to the present day, the library has grown quite a bit in its strategy and approach to include the above core values. The product vision has evolved to adopt a value proposition with inclusion for a broader cross-functional community.
          </p>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container"
        data-testid="pageGrid"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-teamHeader-458 MuiGrid-item MuiGrid-grid-xs-11"
          data-testid="teamHeader"
        >
          <h1
            className="MuiTypography-root MuiTypography-h1"
          >
            Our product team
          </h1>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-459 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            The 2020 pandemic led to forming a small Global remote product team for improving the library and delivering upon the growth strategy. This cross-functional group included product management, product design, and engineering to bring a balanced approach for continuous discovery and continuous delivery. 
          </p>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-459 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <img
            alt="OPL Product Team"
            height="auto"
            src="opl-product-team.png"
            width="auto"
          />
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-459 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            There was a focus on bringing the team together culturally and using various activities for getting to know one another. To align with a process that meets the most user needs the team based discovery practices on human-centered design methodologies with regular user feedback cycles for informing design and product decision making.
          </p>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-459 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            The development approach included pulling from [open source technology, middleware, test-driven, Agile, devops] because... liquam id diam maecenas ultricies mi eget mauris. Odio tempor orci dapibus ultrices in. Vulputate mi sit amet mauris commodo quis imperdiet massa tincidunt. Suspendisse faucibus interdum posuere lorem ipsum.
          </p>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container"
        data-testid="pageGrid"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-teamHeader-460 MuiGrid-item MuiGrid-grid-xs-11"
          data-testid="processModel"
        >
          <h1
            className="MuiTypography-root MuiTypography-h1"
          >
            Our process model
          </h1>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-461 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            The 2020 pandemic led to forming a small Global remote product team for improving the library and delivering upon the growth strategy. This cross-functional group included product management, product design, and engineering to bring a balanced approach for continuous discovery and continuous delivery.
          </p>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-461 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <iframe
            allowFullScreen={true}
            height="600rem"
            src="https://www.youtube.com/embed/N4mBIZg8MnQ"
            title="Open Practice Library"
            width="100%"
          />
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-461 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="https://mobiusloop.com/"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            <p
              className="MuiTypography-root MuiTypography-body1"
            >
              <svg
                height="17"
                viewBox="0 0 17 17"
                width="17"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#2E2E2E"
                >
                  <path
                    d="M15.099 8.79V12.748c0 .264.007.528-.027.79l.034-.252c-.036.254-.102.5-.201.737l.095-.226c-.099.23-.224.446-.376.644l.148-.192c-.152.196-.329.372-.524.524l.191-.148c-.2.152-.414.28-.644.376.076-.032.152-.065.226-.095-.235.099-.482.165-.737.201l.253-.034c-.397.052-.811.027-1.21.027H4.888c-.345 0-.691.002-1.035 0-.133 0-.264-.01-.395-.027l.253.034c-.255-.036-.5-.102-.737-.2L3.2 15c-.23-.099-.447-.224-.644-.376l.192.148c-.196-.152-.373-.328-.525-.524.05.064.1.127.148.192-.151-.2-.279-.414-.376-.644.033.076.065.152.095.226-.098-.236-.165-.483-.201-.737l.034.253c-.051-.397-.026-.811-.026-1.21v-2.272-2.788V4.89c0-.346-.002-.692 0-1.035 0-.133.01-.264.026-.395l-.034.252c.036-.254.103-.5.201-.737l-.095.226c.1-.23.225-.446.376-.644l-.148.192c.152-.195.329-.372.525-.524l-.192.148c.2-.152.414-.279.644-.376-.076.032-.152.065-.226.095.235-.099.482-.165.737-.201l-.253.034c.23-.03.462-.027.693-.027h4.056c.243 0 .5-.106.672-.277.166-.165.29-.435.278-.672-.012-.245-.091-.5-.278-.673C8.693.107 8.465 0 8.207 0H3.87c-.623 0-1.217.152-1.774.425C1.473.731.946 1.24.578 1.823c-.38.6-.575 1.322-.577 2.03-.002.267 0 .533 0 .8v8.492c.002.67.178 1.343.513 1.924.68 1.186 1.98 1.926 3.343 1.93.294.002.589 0 .883 0h8.206c.084 0 .17.002.253 0 .709-.013 1.436-.218 2.029-.611.575-.382 1.079-.91 1.373-1.54.255-.55.399-1.118.399-1.725v-1.69-2.244-.397c0-.243-.106-.5-.277-.672-.166-.166-.435-.289-.673-.278-.245.012-.5.092-.672.278-.171.184-.28.412-.28.67zM11.932 2h4.136c.238 0 .49-.112.66-.292.161-.174.282-.458.271-.708-.01-.258-.09-.526-.272-.708-.182-.18-.406-.292-.66-.292H11.933c-.238 0-.49.112-.66.292-.161.174-.282.458-.271.708.01.258.09.526.272.708.182.178.408.292.66.292z"
                  />
                  <path
                    d="M17 5.068v-1.4-2.227-.509c0-.238-.112-.49-.292-.66C16.534.112 16.25-.01 16 .002c-.258.01-.526.09-.708.272-.18.182-.292.406-.292.66V5.067c0 .238.112.49.292.66.174.161.458.282.708.271.258-.01.526-.09.708-.272.178-.182.292-.408.292-.66z"
                  />
                  <path
                    d="M9.547 8.735l.72-.72 1.714-1.713 2.073-2.073 1.797-1.797c.29-.29.585-.578.872-.871l.012-.013C16.9 1.383 17 1.14 17 .907c0-.223-.098-.488-.265-.642-.172-.157-.398-.275-.641-.264-.241.01-.467.092-.641.264l-.72.72-1.714 1.713-2.073 2.073L9.15 6.568c-.29.29-.585.578-.872.871l-.012.013C8.1 7.617 8 7.86 8 8.093c0 .223.098.488.265.642.172.157.398.275.641.264.241-.009.467-.09.641-.264z"
                  />
                </g>
              </svg>
               
              Learn more on mobiusloop.com
            </p>
          </a>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-461 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            The Open Practice Library utilizes the sections of this process model for tagging and organizing practices within the application. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Semper eget duis at tellus. Enim facilisis gravida neque convallis a cras. Bibendum enim facilisis gravida neque. Vitae et leo duis ut diam. Senectus et netus et malesuada fames ac.
          </p>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container"
        data-testid="pageGrid"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-teamHeader-468 MuiGrid-item MuiGrid-grid-xs-11"
          data-testid="findMentor"
        >
          <h1
            className="MuiTypography-root MuiTypography-h1"
          >
            Find a mentor & more
          </h1>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          >
            We are dedicated to our core values and seek to connect directly with the Open Practice Library community. We love engaging in conversation about the practices and are looking to develop a mentorship program, formulate a beta group, and are always recruiting users for testing needs.
          </p>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="https://openpracticelibrary.slack.com/"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            <img
              alt="Logo"
              src="slack.png"
              width="400rem"
            />
          </a>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root makeStyles-paraSpacer-469 MuiGrid-item MuiGrid-grid-xs-11"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
            href="https://openpracticelibrary.slack.com/"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            <p
              className="MuiTypography-root MuiTypography-body1"
            >
              <svg
                height="17"
                viewBox="0 0 17 17"
                width="17"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="#2E2E2E"
                >
                  <path
                    d="M15.099 8.79V12.748c0 .264.007.528-.027.79l.034-.252c-.036.254-.102.5-.201.737l.095-.226c-.099.23-.224.446-.376.644l.148-.192c-.152.196-.329.372-.524.524l.191-.148c-.2.152-.414.28-.644.376.076-.032.152-.065.226-.095-.235.099-.482.165-.737.201l.253-.034c-.397.052-.811.027-1.21.027H4.888c-.345 0-.691.002-1.035 0-.133 0-.264-.01-.395-.027l.253.034c-.255-.036-.5-.102-.737-.2L3.2 15c-.23-.099-.447-.224-.644-.376l.192.148c-.196-.152-.373-.328-.525-.524.05.064.1.127.148.192-.151-.2-.279-.414-.376-.644.033.076.065.152.095.226-.098-.236-.165-.483-.201-.737l.034.253c-.051-.397-.026-.811-.026-1.21v-2.272-2.788V4.89c0-.346-.002-.692 0-1.035 0-.133.01-.264.026-.395l-.034.252c.036-.254.103-.5.201-.737l-.095.226c.1-.23.225-.446.376-.644l-.148.192c.152-.195.329-.372.525-.524l-.192.148c.2-.152.414-.279.644-.376-.076.032-.152.065-.226.095.235-.099.482-.165.737-.201l-.253.034c.23-.03.462-.027.693-.027h4.056c.243 0 .5-.106.672-.277.166-.165.29-.435.278-.672-.012-.245-.091-.5-.278-.673C8.693.107 8.465 0 8.207 0H3.87c-.623 0-1.217.152-1.774.425C1.473.731.946 1.24.578 1.823c-.38.6-.575 1.322-.577 2.03-.002.267 0 .533 0 .8v8.492c.002.67.178 1.343.513 1.924.68 1.186 1.98 1.926 3.343 1.93.294.002.589 0 .883 0h8.206c.084 0 .17.002.253 0 .709-.013 1.436-.218 2.029-.611.575-.382 1.079-.91 1.373-1.54.255-.55.399-1.118.399-1.725v-1.69-2.244-.397c0-.243-.106-.5-.277-.672-.166-.166-.435-.289-.673-.278-.245.012-.5.092-.672.278-.171.184-.28.412-.28.67zM11.932 2h4.136c.238 0 .49-.112.66-.292.161-.174.282-.458.271-.708-.01-.258-.09-.526-.272-.708-.182-.18-.406-.292-.66-.292H11.933c-.238 0-.49.112-.66.292-.161.174-.282.458-.271.708.01.258.09.526.272.708.182.178.408.292.66.292z"
                  />
                  <path
                    d="M17 5.068v-1.4-2.227-.509c0-.238-.112-.49-.292-.66C16.534.112 16.25-.01 16 .002c-.258.01-.526.09-.708.272-.18.182-.292.406-.292.66V5.067c0 .238.112.49.292.66.174.161.458.282.708.271.258-.01.526-.09.708-.272.178-.182.292-.408.292-.66z"
                  />
                  <path
                    d="M9.547 8.735l.72-.72 1.714-1.713 2.073-2.073 1.797-1.797c.29-.29.585-.578.872-.871l.012-.013C16.9 1.383 17 1.14 17 .907c0-.223-.098-.488-.265-.642-.172-.157-.398-.275-.641-.264-.241.01-.467.092-.641.264l-.72.72-1.714 1.713-2.073 2.073L9.15 6.568c-.29.29-.585.578-.872.871l-.012.013C8.1 7.617 8 7.86 8 8.093c0 .223.098.488.265.642.172.157.398.275.641.264.241-.009.467-.09.641-.264z"
                  />
                </g>
              </svg>
                
              Join the conversation
            </p>
          </a>
        </div>
      </div>
      <div
        className="MuiGrid-root makeStyles-feedback-471 MuiGrid-container"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        />
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1"
        >
          <div
            className="MuiBox-root MuiBox-root-475 makeStyles-feedbackIcon-474"
          >
            <img
              alt="Feedback"
              src="about-feedback.png"
            />
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7"
        >
          <div
            className="MuiBox-root MuiBox-root-476 makeStyles-feedbackBox-470"
          >
            <div
              className="MuiBox-root MuiBox-root-477"
            >
              <h1
                className="MuiTypography-root MuiTypography-h1"
                data-testid="feedbackMsg"
              >
                Please give us feedback
              </h1>
              <h5
                className="MuiTypography-root makeStyles-learn-473 MuiTypography-h5"
              >
                We are human-centered at our core and rely on user feedback for product iterations.
              </h5>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3"
        >
          <div
            className="MuiBox-root MuiBox-root-478"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-feedbackButton-472 MuiButton-disableElevation"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                Share your thoughts
                 
                <svg
                  height="18"
                  viewBox="0 0 23 18"
                  width="23"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M15.912 9.718l-6.017 6.881c-.4.478-1.117.534-1.604.14-.487-.393-.545-1.095-.144-1.572 0 0 0-.028.03-.028l4.354-5.028H2.646c-.63 0-1.146-.506-1.146-1.124 0-.618.516-1.123 1.146-1.123h9.885L8.12 2.836c-.401-.478-.344-1.18.143-1.573s1.175-.337 1.605.112l6.017 6.882c.372.393.372 1.04.028 1.46z"
                    fillRule="evenodd"
                  />
                </svg>
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
